ts/src/main/scala/ch/epfl/ts/component/fetch/Fetch.scala

```diff
@@ -52,10 +52,16 @@ class PullFetchListComponent[T: ClassTag](f: PullFetch[T]) extends Component {
 }
 
 /* Actor implementation */
+/**
+ * To implement your own PushFetchComponent:
+ * 1. Create your class C, C extends PushFetchComponent
+ * 2. Inside your class use callback(what you want to send) 
```

Comment:
In this case you see that on line 60 (inside `PushFetchComponent`), the callback function is already defined as a `send` to all registered actors. This sounds very similar to what I was suggesting earlier (writing a callback function which sends to all actors who are interested).